Jenkinsfile

```diff
@@ -1,24 +1,23 @@
 def mavenVersion='3.3.9'
+node {
+    slave {
+        withOpenshift {
+                withMaven(mavenImage: "maven:${mavenVersion}", 
+		envVars: [ containerEnvVar(key:'GITHUB_OAUTH_CLIENT_ID', value: "${env.GITHUB_OAUTH_CLIENT_ID}"), containerEnvVar(key:'GITHUB_OAUTH_CLIENT_SECRET', value: "${env.GITHUB_OAUTH_CLIENT_SECRET}") ], 
```

Comment:
The env vars are evaluated by the jenkins master. The result is passed to maven container so that they can be used to setup the test environment.